src/notation/view/percussionpanel/percussionpanelpadlistmodel.cpp

```diff
@@ -102,12 +102,12 @@ void PercussionPanelPadListModel::deleteRow(int row)
 void PercussionPanelPadListModel::removeEmptyRows()
 {
     bool rowsRemoved = false;
-    const int lastRowIndex = numPads() / NUM_COLUMNS - 1;
-    for (int i = lastRowIndex; i >= 0; --i) {
-        const int numRows = numPads() / NUM_COLUMNS;
-        const bool rowIsEmpty = numEmptySlotsAtRow(i) == NUM_COLUMNS;
+    const size_t lastRowIndex = numPads() / numColumns() - 1;
```

Comment:
Perhaps numColumns() == 1 [should be considered invalid as well](https://github.com/musescore/MuseScore/pull/28392/files#r2153918506)